src/domain/app/footer/Footer.tsx

```diff
@@ -16,13 +17,26 @@ const footerTheme = {
 const FooterSection: FunctionComponent = () => {
   const { t } = useTranslation();
   const locale = useLocale();
+  const { pathname } = useLocation();
 
   // override Footer component default behaviour which focuses skip-link
   const handleBackToTop = () => {
     window?.scrollTo({ top: 0 });
     document.querySelector<HTMLDivElement>(`#${resetFocusId}`)?.focus();
   };
 
+  const getCategoriesRoute = () => {
+    if (pathname.startsWith(`/${locale}${ROUTES.COURSES}`)) {
+      return ROUTES.COURSES;
+    } else if (pathname.startsWith(`/${locale}${ROUTES.EVENTS}`)) {
+      return ROUTES.EVENTS;
+    } else {
+      return '';
```

Comment:
Maybe it would be more clearer if this just returned null or false when there is no category route

Reply:
This could be implemented this way:

```
  const getCategoriesRoute = () => {
    return [ROUTES.COURSES, ROUTES.EVENTS].find((route) =>
      pathname.startsWith(`/${locale}${route}`)
    );
  };
```

Reply:
fixed